drivers/sensor/qdec_stm32/qdec_stm32.c

```diff
@@ -25,13 +25,18 @@
 
 LOG_MODULE_REGISTER(qdec_stm32, CONFIG_SENSOR_LOG_LEVEL);
 
+#define MODE_X2_TI1  0
+#define MODE_X2_TI2  1
+#define MODE_X4_TI12 2
```

Comment:
Why redefining those here ?

Reply:
I don´t like but I wasn´t sure if it was good practicse to include stuff from the bindings.